retention_dashboard/static/retention_dashboard/js/App.vue

```diff
@@ -2,16 +2,17 @@
   <b-container id="app" class="rd-content-container" fluid>
     <b-row class="rd-app-banner" role="banner">
       <b-col class="col-10 col-md-8">
-        <h1 class="rd-app-title">
+        <h1 class="rd-app-title sr-only">
           <span>Retention Analytics Dashboard</span>
         </h1>
+        <a href="/" class="nav-link p-0 rd-action-link"><img src="/static/retention_dashboard/img/RAD-logo.png" alt="RAD Logo" class="rd-logo"></a>
       </b-col>
       <b-col class="rd-login-info col-2 col-md-4">
         <h3 id="rd_login_header" class="sr-only">
           Your information
         </h3>
-        <span aria-labelledby="rd_login_header" class="hide-lg">Welcome, </span><span id="netid">{{ netid }}</span>
-        <a href="/saml/logout" tabindex="0" class="rd-logout-link">Sign out</a>
+        <span aria-labelledby="rd_login_header" class="hide-lg"><b-icon icon="person-circle" /></span><span id="netid"> {{ netid }}</span> /
```

Comment:
Do you have a trailing "/" on this line? Looks like it here.

Reply:
@jcivjan when you get a chance, mind doing a quick final review?